DependencyInjection/Compiler/FormFactoryCompilerPass.php

```diff
@@ -41,6 +41,7 @@ public function process(ContainerBuilder $container)
 
         // get factories
         $original = $container->getDefinition('form.extension');
+        parent::process($container);
```

Comment:
unfortunately i have no idea anymore how i produced this. in the cmf sandbox we don't see errors atm, so i guess we can leave it, if it makes sense semantically.